<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnapMergeWindow</class>
 <widget class="QMainWindow" name="SnapMergeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SnapMerge</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="toolbarLayout">
      <item>
       <widget class="QPushButton" name="add_files_btn">
        <property name="text">
         <string>Add files…</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="add_folder_btn">
        <property name="text">
         <string>Add folder…</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="include_subfolders_chk">
        <property name="text">
         <string>Subfolders</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="remove_btn">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clear_btn">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacer_toolbar_1">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="move_up_btn">
        <property name="text">
         <string>Move up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="move_down_btn">
        <property name="text">
         <string>Move down</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacer_toolbar_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="sort_name_btn">
        <property name="text">
         <string>Sort by name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sort_type_btn">
        <property name="text">
         <string>Sort by type</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="files_table">
      <property name="editTriggers">
       <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDropMode::InternalMove</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
      </property>
      <property name="dropIndicatorShown" stdset="0">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>#</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Size</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Pages</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Path</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="bottomLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_output">
        <property name="text">
         <string>Destination file:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="browse_output_btn">
        <property name="text">
         <string>Browse…</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="4">
       <layout class="QHBoxLayout" name="runLayout">
        <item>
         <widget class="QLabel" name="total_pages_label">
          <property name="text">
           <string>Total Pages: 0</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer_run">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="run_btn">
          <property name="text">
           <string>Merge</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancel_btn">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="output_line"/>
      </item>
      <item row="1" column="3">
       <widget class="QCheckBox" name="overwrite_chk">
        <property name="text">
         <string>Overwrite if exists</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="log_text">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="merge_progress_bar">
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
